articles/integrations/azure-api-management/configure-azure.md

```diff
@@ -0,0 +1,172 @@
+
+## Configure Azure
+
+::: note
+To complete this, tutorial, you will need to have an account that grants you access to Microsoft's [Azure Portal](https://portal.azure.com).
+:::
+
+In this section, you'll begin by creating an API management instance, as well as the API to manage your cookies application.
+
+### Create Your API Management Instance
+
+To create your API management instance, log into the Azure Portal.
+
+![](/media/articles/integrations/azure-api-mgmt/azure/azure-portal-home.png)
+
+To create a new API management service, click on **New** > **Web + Mobile** > **API management**.
+
+![](/media/articles/integrations/azure-api-mgmt/azure/azure-portal-api-management.png)
+
+You'll be asked to provide the following configuration variables:
+
+| Parameter | Description |
+| --------- | ----------- |
+| Name | The name for your service (which will also be used to create the URL you need to access the service) |
+| Subscription | The Azure subscription plan with which you'll use with the service |
+| Resource group | The collection of [resources](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-portal) sharing a lifecycle, permissions, and policies. You can use an existing resource group or you can create a new one (you'll need to provide a name for the group if you create a new one) |
+| Location | Choose the location that services your API instance |
+| Organization name | The name of your organization |
+| Administrator email | The email address of the person who will be administering this instance |
+| Pricing tier | The pricing tier you want, which determines the number of calls you can make to your API, as well as the maximum amount of data transfer allowed |
+
+![](/media/articles/integrations/azure-api-mgmt/azure/api-mgmt-service-config.png)
+
+Click **Create** to begin provisioning your service.
+
+![](/media/articles/integrations/azure-api-mgmt/azure/deployment-in-progress.png)
+
+### Import Your API
+
+::: note
```

Comment:
I don't think you need a note for this. Leave it as the first sentence of your paragraph, it's basic info, not additional info.